docs/_templates/index.html

```diff
@@ -217,21 +217,128 @@ <h2 class="featurette-heading">One log, many analyses. <span class="text-muted">
                         in Memray - there are many more to choose from, allowing
                         you to view your data in the way that makes the most
                         sense for your specific needs.
-                    </p>.
+                    </p>
                 </div>
                 <div class="col-md-5">
                     <img class="featurette-image img-fluid mx-auto reveal_l"
                         src="{{ pathto('_static/images/stats_example.png', 1) }}" alt="Many reporters">
                 </div>
             </div>
 
-            <hr class="featurette-divider">
-
+            <div class="row featurette">
+                <h2 class="featurette-heading">Memray in the media</h2>
+                <div class="row">
+                    <br>
+                    <div class="card-columns">
```

Comment:
Has much thought been given to the sort order of these cards? If not, perhaps we should think about arranging them by recency?

Reply:
Column major, it appears. Which makes sense, the boxes aren't of a uniform height, so it can't reasonably be row major.